comps/prompt_registry/mongo/prompt.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from typing import Optional
+
+from mongo_store import PromptStore
+from pydantic import BaseModel
+
+from comps.cores.mega.micro_service import opea_microservices, register_microservice
+
+
+class PromptCreate(BaseModel):
+    """This class represents the data model for creating and storing a new prompt in the database.
+
+    Attributes:
+        prompt_text (str): The text content of the prompt.
+        user (str): The user or creator of the prompt.
+    """
+
+    prompt_text: str
+    user: str
+
+
+class PromptId(BaseModel):
+    """This class represent the data model for retrieve prompt stored in database.
+
+    Attributes:
+        user (str): The user of the requested prompt.
+        prompt_id (str): The prompt_id of prompt to be retrieved from database.
+    """
+
+    user: str
+    prompt_id: Optional[str] = None
+    prompt_text: Optional[str] = None
+
+
+@register_microservice(
```

Comment:
We will support multiple endpoints within the same microservice. Starting a separate microservice for each endpoint is not practical.

Reply:
This current design is referenced from other components like dataprep design for now to keep aligned and avoid confusion. For multiple endpoint within same microservices, we can make appropriate changes when it is supported across. Is that ok?

Reply:
@lvliang-intel , are there still  any concern on this comment? Can you please check it.